docker/linux-image.sh

```diff
@@ -83,7 +83,12 @@ main() {
             ;;
         x86_64)
             arch=amd64
-            kernel="${kversion}-amd64"
+            #kernel="${kversion}-amd64"
```

Comment:
While building an image with ubuntu 20.04 it fails to find some packages. Adding this allows it to find them, and the kernel bump was necessary for the image to build locally. Also, as far as I can see there is only one docker image using amd64 that is using this script so these changes will have no effect on other images